modules/machineset-creating.adoc

```diff
@@ -0,0 +1,261 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id='machineset-creating-{context}']
+= Creating a MachineSet
+
+You can create more MachineSets. Because the MachineSet definition contains
+details that are specific to the AWS region that the cluster is deployed in,
+you copy an existing MachineSet from your cluster and modify it.
+
+.Prerequisites
+
+* Deploy an {product-title} cluster.
+* Install the `oc` command line and log in as a user with <some permissions>.
+
+.Procedure
+
+. View the current MachineSets.
++
+[source,bash]
+----
+$ oc get machinesets -n openshift-cluster-api
+
+NAME                         DESIRED   CURRENT   READY     AVAILABLE   AGE
+190125-3-worker-us-west-1b   2         2         2         2           3h
+190125-3-worker-us-west-1c   1         1         1         1           3
+----
+
+. Export the source of a MachineSet to a text file:
++
+[source,bash]
+----
+$ oc get machineset <machineset_name> -n \
+     openshift-cluster-api -o yaml > <file_name>.yaml
+----
++
+In this command, `<machineset_name>` is the name of the current MachineSet that
+is in the AWS region you want to place your new MachineSet in, such
+as `190125-3-worker-us-west-1c`, and `<file_name>` is the name of your new
+MachineSet definition.
+
+. Update the `metadata` section of `<file_name>.yaml`:
++
```

Comment:
@kalexand-rh I just tried this procedure using the latest 1.14 installer. I did work, apparently. However, many of the fields in this file are different, in case you weren't aware.